airflow/api_connexion/openapi/v1.yaml

```diff
@@ -377,6 +377,12 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
+        - name: only_active
+          in: query
+          schema:
+            type: boolean
+          required: false
+          description: Only return DAGs which were seen on the last DagBag load.
```

Comment:
This isn't quite true -- we'll need to reword it. Something like "DAGs which are currently seen by the schedulers"

Reply:
Sounds good - I'll update the comment in the DAG model as well, since this is where I got this description. 
https://github.com/apache/airflow/blob/0d366c1f297ec25b5f8e01c6d72a312e2ee64d4d/airflow/models/dag.py#L2069